airflow/models/dagrun.py

```diff
@@ -1155,7 +1155,7 @@ def schedule_tis(self, schedulable_tis: Iterable[TI], session: Session = NEW_SES
 
         Each element of ``schedulable_tis`` should have it's ``task`` attribute already set.
 
-        Any EmptyOperator without callbacks is instead set straight to the success state.
+        Any EmptyOperator without callbacks or outlets is instead set straight to the success state.
```

Comment:
what about `inlets`?

Reply:
Nothing actually happens with inlets, so I think we can ignore those. (Dataset outlets, however, result in dataset events, and possibly run queue population)